resources/assets/less/bem/beatmap-icon.less

```diff
@@ -80,6 +80,13 @@
     }
   }
 
+  &--with-tooltip {
+    &:hover::before {
+      box-shadow: 0 0 0 4px #333333;
```

Comment:
due to alignment problem, the effect for this is changed. This outline thing is removed and `__shadow` changes to `.thicker-box-shadow()` on hover instead.